### DISCUSSION TITLE
Notification Center Redesign.

### DISCUSSION THREAD

**Discussion by evvvrado** [monkeytypegame/monkeytype#4688] [Ideas]
As I was thinking about how to get involved and contribute to the MonkeyTypes community, I couldn't help but notice the Notification Center's layout on the platform. It's working pretty well and looks good, but I think it has the potential to be even better. That's why I'd like to take on the challenge of redesigning the Notification Center. What do you guys think?

<img src="https://github.com/monkeytypegame/monkeytype/assets/76765966/5c1b5ced-a5ab-4af9-ae8b-60f5b4dcfc1d" alt="Current Notify Center Layout" width="300">

**Comment by Miodec** [comment]
Feel free to submit some sort of preview or what youve got in mind.

**Reply by evvvrado** [reply]
I intend to maintain the core functionality of the TypeScript Notification Center while focusing on its appearance. My main objective is to create a universal Notify Card that can be used across all themes, with only a distinction between the types: "bad", "good" and "notice".

## Notice.
![image](https://github.com/monkeytypegame/monkeytype/assets/76765966/ce79815b-0161-4138-a3d7-34ea45b1fa11)
<br/>
<br/>
## Good.
![image](https://github.com/monkeytypegame/monkeytype/assets/76765966/a33fb2e4-3d19-4ecf-8ad8-ed0f1c793143)
<br/>
<br/>
## Success.
![image](https://github.com/monkeytypegame/monkeytype/assets/76765966/5f1f533f-1cf5-456e-9a2c-ecd7a57f64a1)

**Reply by Miodec** [reply]
Im not sure about moving away from the currently active theme colors to a set green/blue/red color scheme. It depends though, maybe I will like it when i actually get to use it.

**Reply by evvvrado** [reply]
I totally agree with you! I think we'll only really know if it's a good idea once we give it a shot. The reason I think it might be important is that some of the themes we have right now have problems with color contrast. So, having a universal color scheme could potentially solve these issues. 

### Some themes with contrast issues:
<img width="378" alt="Captura de Tela 2023-10-03 às 09 24 15" src="https://github.com/monkeytypegame/monkeytype/assets/76765966/cbbceaab-55cd-4392-ac07-a2319810e1fa">
<img width="378" alt="Captura de Tela 2023-10-03 às 09 24 33" src="https://github.com/monkeytypegame/monkeytype/assets/76765966/68d26b4f-e6ba-4565-9eaa-8b8f88eba679">
<img width="378" alt="Captura de Tela 2023-10-03 às 09 24 49" src="https://github.com/monkeytypegame/monkeytype/assets/76765966/54eb6d94-5379-402d-bef9-015ea7090ed9">
<img width="378" alt="Captura de Tela 2023-10-03 às 09 26 44" src="https://github.com/monkeytypegame/monkeytype/assets/76765966/28716597-13bc-4b14-bcd5-1cadda5c88c6">
